### ISSUE TITLE
`himalaya flags` panics

### ISSUE THREAD

**Issue by cschmatzler** [soywod/himalaya#190]
Running the `flags` command, himalaya panics no matter what other arguments I pass.

Version 0.4.0 downloaded from GitHub Releases

```
❯ RUST_BACKTRACE=1 himalaya flags add 1 Seen
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/msg/cli.rs:186:54
stack backtrace:
   0: rust_begin_unwind
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/panicking.rs:92:14
   2: core::panicking::panic
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/panicking.rs:50:5
   3: himalaya::msg::cli::msg_matches_list
   4: himalaya::msg::cli::msg_matches
   5: himalaya::main
```

```
❯ cat /etc/os-release 
NAME="void"
ID="void"
DISTRIB_ID="void"
PRETTY_NAME="void"
```

```
❯ uname -r
5.11.22_1
```

**Comment by soywod** [comment]
The flags command seems to be badly interpretted (the error points to the list command). I will investigate and let you know!

**Comment by soywod** [comment]
This issue should be fixed by the incomming refactor. I let you know!

**Comment by soywod** [comment]
Fixed in the [`v0.5.0`](https://github.com/soywod/himalaya/releases/tag/v0.5.0) :tada:
